### PR TITLE
Adding "Firmeza" as dialect from Sao Paulo.

### PR DESCRIPTION
**Pull request by juliodias**
## Pre-Submission Checklist

- You have only one commit (if not, squash them into one commit).
- All new and existing tests pass the command `yarn test` and `yarn lint`.

## Description

Adding "Firmeza" as dialect from Sao Paulo.

### CODE REVIEW

File: dialects/paulistes.json [hunk 1]
```diff
@@ -200,6 +200,18 @@
       "Descolei um emprego novo, começarei amanhã",
       "Descola aquela grana para mim"
       ]
+  }, 
+  {
+    "dialect": "Firmeza",
+    "meanings": [
+      "Gente boa",
+      "Tudo tranquilo?",
+      "Firmeza e você?"
```
Comment (mvfsillva) [inline dialects/paulistes.json]: I think this meaning is: `Tudo tranquilo e você?`

**Comment by mvfsillva** [comment]
@juliodias  could you resolve the conflict?

**Comment by juliodias** [comment]
> @juliodias could you resolve the conflict?

Hi @mvfsillva, I fixed the conflicts, can you validate, please?

**Comment by mvfsillva** [comment]
Why closed the issue @juliodias?

**Pull request closed without merging**